### ISSUE TITLE
Permission to add Gatsby monorepo blog post to blog post list. 

### ISSUE THREAD

**Issue by andrico1234** [gatsbyjs/gatsby#9876]
## Summary

I've previously published [this](https://codeburst.io/develop-and-deploy-your-own-react-monorepo-app-in-under-2-hours-using-lerna-travis-and-now-2b140d647238) blog post and felt like it would be a useful resource to add to the list of [Gatsby blog](https://www.gatsbyjs.org/blog/) posts. 

## Relevant information

The tutorial covers the process of converting a regular Gatsby application into a Monorepo, builds it to a static site then deploys it to Now using Travis.

If you feel this article doesn't fit the Gatsby Blog, then I'd love to know how I could improve it so it becomes appropriate.

Thanks!

**Comment by andrico1234** [comment]
@kakadiadarpan Is there an update to this?

**Comment by jgierer12** [comment]
You might have more luck directly contacting the Gatsby team at [@gatsbyjs on Twitter](https://twitter.com/gatsbyjs)

**Comment by DSchau** [comment]
@jreidgreer I think we probably prefer issues so that they're visible/in the open, so that others can comment and take a look.

@andrico1234 I wouldn't say there's anything you need permission for! I would highly recommend that you open the PR with the blog post added, and then we can discuss further about getting it merged--if that makes sense. We don't really have a formalized intake process, it's more a matter of ensuring that blog posts are generally valuable for the community.

Does that sound OK? Any questions here?

**Comment by KyleAMathews** [comment]
It'd be a lot of work to convert the post to markdown so we should make the decision before he does a PR.

**Comment by andrico1234** [comment]
As soon as I get the go-ahead from one of the Gatsby team, I'll be happy to open up a PR. 

Unless there's some glaring misinformation in my article, i think a tutorial in setting up a monorepo with some external tools is a super useful one, considering we did something similar in the Pride In London website.

**Comment by andrico1234** [comment]
@DSchau  @kakadiadarpan Any update on this?

@jgierer12 I contacted the team on Twitter first and was told to open up a PR as I have done. 

It seems like there's no ownership over this PR, so can one of the Gatsby team decide amongst themselves as someone who would be happy to make the decision as to whether or not I can add the article to the blog? It's been over a month and nothing's progressed.

**Comment by DSchau** [comment]
@andrico1234 you're right - sorry for the delay here.

I think the blog post is a great introduction to some disparate things _around_ Gatsby, e.g. CI integration, Enzyme, and monorepos. I do wish it was a little more focused on Gatsby specifically, so any additional work to spin it that direction would be appreciated, but not strictly speaking necessary. The blog post, as authored, is great and worthy of inclusion.

There is some precedent here of less-Gatsby focused blog posts (e.g. see [Deploying to Azure](https://www.gatsbyjs.org/blog/2018-11-05-deploying-gatsby-to-azure/), so I think this is worthwhile to get a PR up and convert it to Markdown.

If you're willing to take the time to convert the post, validate how it looks with Markdown and with the local development server, we'd love the post on our blog.

TL;DR So sorry for the delay, and we absolutely want your blog post on our blog if you're willing to do the PR and convert to Markdown.

💜

**Comment by andrico1234** [comment]
@DSchau @jgierer12 

I've got a PR with my post on it, it looks fine after running it locally

https://www.dropbox.com/s/7c5yyayal6v9v05/Screenshot%202018-12-15%2020.39.32.png?dl=0

https://github.com/gatsbyjs/gatsby/pull/10490

**Comment by andrico1234** [comment]
Hey all!

About a month ago I opened the PR (https://github.com/gatsbyjs/gatsby/pull/10490), and a couple of the members of the Gatsby team were kind enough to go through the entire article (which is lengthy) and leave a few suggestions for improvements. 

I fixed them all about 2 weeks ago and have been waiting for approvals. I can understand that those particular reviewers may be busy, or out of office completely, but can someone take ownership of the PR so I can get it merged in?

It's been almost 2 months since I've opened up this github issue and it feels like 2 months (end to end) to get a blog post added to the repo is excessive.

I'm going to tag a few of the people who have been involved with this PR to see if we can get the ball rolling again. 

Cheers. 

@DSchau @shannonbux @calcsam @KyleAMathews
